shark/examples/shark_importer/albert_shark_test.py

```diff
@@ -0,0 +1,51 @@
+# RUN: %PYTHON %s
+import numpy as np
+from shark.shark_importer import SharkImporter, GenerateInputSharkImporter
+
+model_path = "https://tfhub.dev/tensorflow/lite-model/albert_lite_base/squadv1/1?lite-format=tflite"
+
+# for different model, we have to generate different inputs.
+class AlbertInput(GenerateInputSharkImporter):
+    def __init__(self, input_details, model_source_hub):
+        super(AlbertInput, self).__init__(input_details, model_source_hub)
+
+    # Inputs modified to be useful albert inputs.
+    def generate_inputs(self):
+        for input in self.input_details:
+            print("\t%s, %s", str(input["shape"]), input["dtype"].__name__)
+
+        args = []
+        args.append(
+            np.random.randint(low=0, high=256, size=self.input_details[0]["shape"], dtype=self.input_details[0]["dtype"]))
+        args.append(np.ones(shape=self.input_details[1]["shape"], dtype=self.input_details[1]["dtype"]))
+        args.append(np.zeros(shape=self.input_details[2]["shape"], dtype=self.input_details[2]["dtype"]))
+        return args
+
+    def compare_results(self, iree_results, tflite_results, details):
+        print("len(iree_results):", len(iree_results))
+        print("len(tflite_results):", len(tflite_results))
+        assert (len(iree_results) == len(tflite_results), f"Number of results do not match")
+
+        for i in range(len(details)):
+            iree_result = iree_results[i]
```

Comment:
Can we change all the variable name and our functions from "iree" to "shark"?

Reply:
Done